src/main/webapp/timestamps.js

```diff
@@ -39,3 +39,13 @@ function timestampToString(timestampMs) {
 function secondsToMilliseconds(seconds) {
   return Math.round(seconds * MILLISECONDS_PER_SECOND);
 }
+
+/**
+ * Returns {string} A time range from
```

Comment:
Can you do `Returns a time range {string} from ...` to make it flow better? Or is this order required.

Reply:
Just read that part of the styleguide again and it looks like `@return {string} A time range string created from ...` would be more typical

Reply:
The order I have is what I've seen in the [style guide](https://google.github.io/styleguide/jsguide.html#jsdoc-method-and-function-comments) so I think it is better leave it as is.